engine/geolocation.go

```diff
@@ -0,0 +1,31 @@
+// Copyright © 2017 thingful
+//
+// Geolocation minimal js object mimic
+// https://dev.w3.org/geo/api/spec-source.html#position
+
+package engine
+
+import (
+	"time"
+)
+
+type coordinates struct {
+	Latitude         float64 // specified in decimal degrees
+	Longitude        float64 // specified in decimal degrees
+	Accuracy         float64 // specified in meters
+	Altitude         float64 // specified in meters
```

Comment:
Out of interest what happens if you try to access one of these properties - is it just undefined?